src/Pickles/Pickles/ConfigurationReporter.cs

```diff
@@ -35,6 +35,7 @@ public void ReportOn(IConfiguration configuration, Action<string> writeToLog)
             writeToLog($"Language                       : {configuration.Language}");
             writeToLog($"Incorporate Test Results?      : {(configuration.HasTestResults ? "Yes" : "No")}");
             writeToLog($"Include Experimental Features? : {(configuration.ShouldIncludeExperimentalFeatures ? "Yes" : "No")}");
+            writeToLog($"Ignore Tag                       : {configuration.ExcludeTags}");
```

Comment:
Please also change this instance of "Ignore" to Exclude" - and make sure the `:`-signs in the lines are aligned with each other.